client/src/modules/cohort/model/actions.js

```diff
@@ -48,6 +54,74 @@ const updateCohortFailure = err => ({
   payload: err.message
 });
 
+const archiveCohortSuccess = data => ({
+  type: CohortActionTypes.ARCHIVE_SUCCESS,
+  payload: data
+});
+
+const archiveCohortFailure = errMessage => ({
+  type: CohortActionTypes.ARCHIVE_FAILURE,
+  payload: errMessage
+});
+
+const archiveCohortRequest = () => ({
+  type: CohortActionTypes.ARCHIVE_REQUEST
+});
+
+const restoreCohortSuccess = (data, cohortId) => ({
+  type: CohortActionTypes.RESTORE_SUCCESS,
+  payload: { data, cohortId }
```

Comment:
change this to _id so you dont have to map this later on in reducer.
It'is restoreCOhortSucces action creator so it's clear that _id belongs to cohort